src/tlv/TlvFloat.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license
+ * Copyright 2022 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { DataReaderLE } from "../util/DataReaderLE";
+import { DataWriterLE } from "../util/DataWriterLE";
+import { TlvType, TlvCodec, TlvTag } from "./TlvCodec";
+import { TlvSchema } from "./TlvSchema";
+
+const FLOAT_RANGE = { min: -340282346638528859811704183484516925440.0, max: 340282346638528859811704183484516925440.0 };
```

Comment:
Perhaps formally define FLOAT_MAX and FLOAT_MIN and use those to define FLOAT_RANGE?
Seems like FLOAT_MAX and FLOAT_MIN as separate consts may come in handy later.

Reply:
Done